src/components/atoms/TeamCard/index.jsx

```diff
@@ -0,0 +1,33 @@
+import Body3 from '../../shared/Typography/Body3';
+import { Socials, TeamCards, Wrapper } from '../../molecules/TeamCard/styles';
+import { Links } from '../../molecules/TeamCard/styles';
```

Comment:
The styles for index.jsx should be here not in molecules. It will create further confusion.

Reply:
Done